src/libcore/num/f64.rs

```diff
@@ -33,7 +33,11 @@ pub const MANTISSA_DIGITS: u32 = 53;
 #[stable(feature = "rust1", since = "1.0.0")]
 pub const DIGITS: u32 = 15;
 
-/// Difference between `1.0` and the next largest representable number.
+/// [Machine epsilon] value for `f64`.
+///
+/// This is the difference between `1.0` and the next largest representable number.
```

Comment:
For me this does not parse correctly. Epsilon is the number which when added to 1.0 will yield a different number than 1.0. This sounds more correct to me:
> This is the difference between `1.0` and the next smallest representable number.

Reply:
Would it be clearer if it was "next larger representable number"? The usage of "large" is to indicate it's in the positive direction (above 1.0). Maybe it's sufficient to just say "next representable number"?

Reply:
@frewsxcv Yes, both would work for me. I'm not a mathematician but if I remember correctly the direction doesn't even matter.

Reply:
The direction does matter because floating point numbers < 1 have a smaller exponent (due to normalization) and therefore are more dense than those >= 1. The "epsilon" in the other direction would be half of the one in the positive direction.

Reply:
@rkruppe Oh right, I forgot about normalisation. Thanks for the reminder.